main.py

```diff
@@ -208,41 +199,117 @@ def get_flows_to_be_installed(self):
                     # Change table_id before being added
                     flow["flow"].update({"table_id": expected_table_id})
                     install_flows[switch].append(flow["flow"])
+
         log.info(f"of_multi_table pushing flows, pipeline: {pipeline}")
+        self.manage_miss_flows(pipeline, flows_by_swich)
         self.send_flows(delete_flows, "delete")
         self.send_flows(install_flows, "install")
+
         if pipeline.get("status") is None:
             self.pipeline_controller.disabled_pipeline(pipeline_id)
             msg = f"Pipeline {pipeline_id} disabled"
             log.debug(f"of_multi_table result {msg}")
         else:
-            self.install_miss_flows(pipeline)
             self.pipeline_controller.enabled_pipeline(pipeline_id)
             msg = f"Pipeline {pipeline_id} enabled"
             log.debug(f"of_multi_table result {msg}")
 
-    def install_miss_flows(self, pipeline: dict):
+    @staticmethod
+    def get_miss_flows(flows_by_swich: Dict[str, Dict]):
+        """Get miss flows reformated as {"table_id": {flows}} from
+        a single switch"""
+        miss_flows = {}
+        flow_ids = set()
+        # Keys needed to compare miss flow entries, except table_id
+        compare = {"priority", "instructions", "match"}
+        for _, flows in flows_by_swich.items():
+            for flow in flows:
+                owner = flow["flow"].get("owner")
+                if owner == "of_multi_table":
+                    flow = flow["flow"]
+                    miss_flows[flow["table_id"]] = {
+                        key: flow[key] for key in compare if flow.get(key) is not None
+                    }
+                    flow_ids.add(flow["table_id"])
+            # Miss flows are the same in all switches
+            break
+        return miss_flows, flow_ids
+
+    def manage_miss_flows(self, pipeline: Dict, flows_by_swich: Dict[str, Dict]):
+        """Determine whether to install and/or delete miss_flows"""
+        if not flows_by_swich:
+            return
+        miss_table = {}
+        table_ids = set()
+        miss_flows, flow_ids = self.get_miss_flows(flows_by_swich)
```

Comment:
Same as my prior comment regarding `flow_ids` if you could rename it more closely related to `stored_table_ids` or something else that would improve understanding of what it's actually holding.

Reply:
Changed naming.